### PR TITLE
Fixed the near me list to display the correct order

### PR DESCRIPTION
**Pull request by CDavidSamuelsson**
#191

### CODE REVIEW

File: app/src/main/java/com/flexicharge/bolt/activities/MainActivity.kt [hunk 1]
```diff
@@ -408,6 +412,7 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback, ChargePointListAda
         }
     }
 
+    @SuppressLint("SuspiciousIndentation")
```
Comment (foal20ym) [inline app/src/main/java/com/flexicharge/bolt/activities/MainActivity.kt:415]: No need to remove or fix, but try to not use warning suppressors if possible. Also remove any comments you come across like the one below.

File: app/src/main/java/com/flexicharge/bolt/activities/MainActivity.kt [hunk 2]
```diff
@@ -446,15 +451,37 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback, ChargePointListAda
...
+            sortedChargePointsList.addAll(sortedChargePoints)
+
+            val distanceAsString = mutableListOf<String>()
+            distanceAsString.addAll(sortedDistanceToChargePoint.map { it.toString() })
+
+
+
             lifecycleScope.launch(Dispatchers.Main) {
-                listOfChargePointsRecyclerView.adapter = ChargePointListAdapter(remoteChargePoints.value, this@MainActivity, distanceToChargePoint, chargerCount)
+              //  listOfChargePointsRecyclerView.adapter = ChargePointListAdapter(remoteChargePoints.value, this@MainActivity, distanceAsString, sortedChargerCount)
```
Comment (foal20ym) [inline app/src/main/java/com/flexicharge/bolt/activities/MainActivity.kt:478]: Remove commented code for better performance 
```suggestion
```

File: app/src/main/java/com/flexicharge/bolt/activities/MainActivity.kt [hunk 3]
```diff
@@ -446,15 +451,37 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback, ChargePointListAda
...
+                listOfChargePointsRecyclerView.adapter = ChargePointListAdapter(sortedChargePointsList, this@MainActivity, distanceAsString, sortedChargerCount)
+
             }
 
+
+
         }
         //listOfChargePointsRecyclerView.adapter = ChargePointListAdapter(chargePoints.map { it.chargePointAddress }, chargePoints.map {it.chargePointId}, chargePoints.map { it.chargePointId})
```
Comment (foal20ym) [inline app/src/main/java/com/flexicharge/bolt/activities/MainActivity.kt:486]: Remove commented code and other comments you come across 👍 
```suggestion
```

File: app/src/main/java/com/flexicharge/bolt/activities/MainActivity.kt (hunk 1 above)
Comment (CDavidSamuelsson) [reply app/src/main/java/com/flexicharge/bolt/activities/MainActivity.kt:416]: it can be removed, dont know why it is there

**Review by foal20ym — APPROVED** [review]

**Pull request merged**